packages/core/src/tracing/sentrySpan.ts

```diff
@@ -107,6 +107,39 @@ export class SentrySpan implements Span {
     }
   }
 
+  /**
+   * This should generally not be used,
+   * but we need it for being comliant with the OTEL Span interface.
+   *
+   * @hidden
+   * @internal
+   */
+  public addLink(_link: unknown): this {
+    return this;
+  }
+
+  /**
+   * This should generally not be used,
+   * but we need it for being comliant with the OTEL Span interface.
+   *
+   * @hidden
+   * @internal
+   */
+  public addLinks(_links: unknown[]): this {
+    return this;
+  }
+
+  /**
+   * This should generally not be used,
+   * but we need it for being comliant with the OTEL Span interface.
+   *
+   * @hidden
+   * @internal
+   */
+  public recordException(_exception: unknown, _time?: number | undefined): void {
```

Comment:
comment/unactionable: "recordException" is a very unfortunate function name  for users seeing this method when interacting with the Sentry SDK 😬 

(ignore this comment, I'm just fairly sure someone is gonna call it "accidentally" instead of `captureException`)

Reply:
Had the same thought! I had an idea that we could alias this to captureException. Maybe we should also make it very clear in the JS doc that captureException needs to be preferred.

Reply:
> I had an idea that we could alias this to captureException

yeah this also crossed my mind. But I'd rather do it when it comes up to save on bundle size.